### DISCUSSION TITLE
Managing large content structures

### DISCUSSION THREAD

**Discussion by okohll** [google/docsy#974] [General]
Hi,

Here's a question a bit removed from the nuts and bolts of working with Docsy, but one I thought some people may have some insight on.

We have a site which is getting relatively large (?) at 113 markdown pages so far in the content folder. Multiple people are starting to work on adding to it and keeping everything up to date.

Are there any tactics or tools people have used to manage the content authoring process? For example allowing us to list which the most or least recently updated pages are, see who's been working on which areas, mark a set of pages which need updating for a particular reason, give some management visibility etc.

Our first thought was some sort of spreadsheet / database, maybe manually updated to start with, tracking folder structure, pages, tags and updates, with later on an integration, possibly with incoming changes from the source github repository.

Does anyone else do anything similar?

Cheers
Oliver

**Comment by fekete-robert** [comment]
Hi, I haven't actually used any of these, but have you tried looking into a git statistics tool, like the ones mentioned here? https://stackoverflow.com/questions/1828874/generating-statistics-from-git-repository

**Reply by okohll** [reply]
Thanks, those might do part of what I'm looking for, will take a look. I guess the rest is less a technology issue than a management one.
